shared/common-adapters/check-circle.tsx

```diff
@@ -23,21 +24,30 @@ const CheckCircle = (props: Props) => {
       evt.stopPropagation()
     }
   }
+  const [hover, setHover] = React.useState(false)
+  const onMouseEnter = () => setHover(true)
```

Comment:
its likely better to just use the hover css styles we already have. you don't need to bookkeep mouse for this

Reply:
How do I make them customizable? Do I add new hover color mappings to the Icon component?

Reply:
is the customization even used

Reply:
there's `selectedColor={Styles.isDarkMode() ? Styles.globalColors.black : undefined}` which only exists in two places
(as a sidenote I'll obviously recheck darkmode before merging, i'm mostly trying to figure out the approach here)

Also black10 goes to black85 on hover using the default classes. Even if I were to implement a custom color mapping table, it will all have to rely on hardcoded CSS. Up to you :)

Reply:
this component should just be dark mode aware and not need this parameter. then the css can be hardcoded (thats fine). we already have the classes so just use `className=".color_black"` etc